cmd/utils/flags.go

```diff
@@ -2011,7 +2011,7 @@ func MakeChainDatabase(ctx *cli.Context, stack *node.Node, readonly bool) ethdb.
 		}
 		chainDb = remotedb.New(client)
 	default:
-		chainDb, err = stack.OpenDatabaseWithFreezer("chaindata", cache, handles, ctx.String(AncientFlag.Name), "", readonly)
+		chainDb, err = stack.OpenDatabaseWithFreezer(ethconfig.DatabaseName, cache, handles, ctx.String(AncientFlag.Name), ethconfig.DatabaseNamespace, readonly)
```

Comment:
I think you can just put the constant metric namespace here. No need to use a global constant var

Reply:
how about we just use the constant name here, @MariusVanDerWijden WDYT

Reply:
No preference

Reply:
I prefer to the fist simple one, keep it simple